.github/workflows/oneshot_candidate_release.yml

```diff
@@ -0,0 +1,56 @@
+name: Schedule candidate release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  tag_release:
+    name: "Tag candidate release"
+    runs-on: ubuntu-18.04
+    # Don't run this in everyone's forks.
+    if: github.repository == 'google/iree' || github.repository == 'NodLabs/SHARK'
```

Comment:
No need for this check with only a `workflow_dispatch` trigger.

In fact, we might be able to use the same workflow and just rework this check to something like `if: repo is google/iree || triggered by workflow dispatch`

GitHub Actions are weird, but we shouldn't have a list of "forked approved for running this workflow". Workflows can be disabled per repository, but we added this condition to avoid superflous builds in forks where there is no need to generate release artifacts on such a schedule (and those builds were failing without a write access token set IIRC).

Reply:
ah makes sense. Will remove.